esp/src/src/ESPLogicalFile.ts

```diff
@@ -237,15 +237,16 @@ var LogicalFile = declare([ESPUtil.Singleton], {    // jshint ignore:line
         }
         this.logicalFile = this;
     },
-    save: function (description, args) {
+    save: function (request, args) {
         //WsDfu/DFUInfo?FileName=progguide%3A%3Aexampledata%3A%3Akeys%3A%3Apeople.lastname.firstname&UpdateDescription=true&FileDesc=%C2%A0123&Save+Description=Save+Description
         var context = this;
         WsDfu.DFUInfo({
             request: {
                 Name: this.Name,
                 Cluster: this.Cluster,
                 UpdateDescription: true,
-                FileDesc: description
+                FileDesc: request.Description,
+                Protect: request.isProtected === true ? 1 : 2
```

Comment:
@wangkx - is this 1 || 2 correct?

Reply:
1 for Protect, 2 for Unprotect, 0 for NoChange